x-pack/plugins/security_solution/public/siem_migrations/rules/service/success_notification.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import type { CoreStart } from '@kbn/core-lifecycle-browser';
+import { i18n } from '@kbn/i18n';
+import {
+  SecurityPageName,
+  useNavigation,
+  NavigationProvider,
+} from '@kbn/security-solution-navigation';
+import type { ToastInput } from '@kbn/core-notifications-browser';
+import { toMountPoint } from '@kbn/react-kibana-mount';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { EuiButtonEmpty, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import type { RuleMigrationTaskStats } from '../../../../common/siem_migrations/model/rule_migration.gen';
+
+type RuleMigrationStats = RuleMigrationTaskStats & { migration_id: string };
+
+export const getSuccessToast = (migration: RuleMigrationStats, core: CoreStart): ToastInput => ({
+  color: 'success',
+  iconType: 'check',
+  title: i18n.translate('xpack.securitySolution.siemMigrations.rulesService.polling.successTitle', {
+    defaultMessage: 'Rule migration completed.',
+  }),
+  text: toMountPoint(
+    <NavigationProvider core={core}>
+      <SuccessToastContent migration={migration} />
+    </NavigationProvider>,
+    core
+  ),
+});
+
+const SuccessToastContent: React.FC<{ migration: RuleMigrationStats }> = ({ migration }) => {
+  const { migration_id: path, rules } = migration;
+  const navigation = { deepLinkId: SecurityPageName.siemMigrationsRules, path };
```

Comment:
We will need to add a route to handle `migration_id` as a path parameter (somewhere here `x-pack/plugins/security_solution/public/siem_migrations/routes.tsx`). Right now we only open a default page where we would pick the first available migration and use it.